spec/models/manageiq/providers/redhat/infra_manager/refresh/refresher_4_1_spec.rb

```diff
@@ -147,7 +147,7 @@ def load_response_mock_for(filename)
     @ems.reload
 
     assert_table_counts(2)
-    assert_ems
+    # TODO: add 'assert_ems' to the assertion below once ems[:api_version] is updated properly by the graph refresh
```

Comment:
I am not fan of such approaches. I think that we should either provide the code which works or remove the assert when it is no longer needed.

Reply:
removed.

Reply:
with this PR, the master will work with v3 and v4, but won't update api_version for ems if using graph refresh.
without this PR, v4 refresh fails (regardless of graph refresh).

I think we should get it in and find a way to handle the ems persistence as part of the graph refresh.